tests/virt/node/general/test_oom.py

```diff
@@ -79,13 +79,15 @@ def _transfer_loop():
         tcp_timeout=TCP_TIMEOUT_30SEC,
     )
 
-    transfer = Process(target=_transfer_loop)
+    stop_event = Event()
+    transfer = Thread(target=_transfer_loop, daemon=True)
```

Comment:
Explicit dependencies make it easier to understand what _transfer_loop() needs, without relying on hidden global variables, this shouldn't block the PR but would be a `good to have`
```suggestion
    transfer = Thread(target=_transfer_loop, args=(stop_event) daemon=True)
```

Reply:
good point, thanks